src/main/java/rs117/hd/overlays/FrameTimer.java

```diff
@@ -115,6 +123,14 @@ public void begin(Timer timer) {
 	}
 
 	public void end(Timer timer) {
+		if (log.isDebugEnabled() && timer.isGpuTimer && plugin.glCaps.OpenGL43) {
+			log.debug("Pop {}", timer, new Throwable());
+			// TODO: Ensure we never pop if we haven't pushed
```

Comment:
Attempting to pop when nothing has been pushed leads to a stack underflow error. Presumably we could fix this by simply checking `activeTimers`, but that annoyingly requires other changes, since we'd need to allow it to reset.